pkg/keyspace/tso_keyspace_group.go

```diff
@@ -712,6 +702,11 @@ func (m *GroupManager) FinishSplitKeyspaceByID(splitTargetID uint32) error {
 	var splitTargetKg, splitSourceKg *endpoint.KeyspaceGroup
 	m.Lock()
 	defer m.Unlock()
+
+	failpoint.Inject("pauseTxn", func() {
+		time.Sleep(2 * time.Second)
```

Comment:
How about using `pause` instead of sleep?

Reply:
@rleungx @ystaticy i 'v replaced sleep with pause, please take a look